Apps/ValidationTests/Win32/Resource.h

```diff
@@ -0,0 +1,30 @@
+//{{NO_DEPENDENCIES}}
+// Microsoft Visual C++ generated include file.
+// Used by App.rc
+
+#define IDS_APP_TITLE			103
+
+#define IDR_MAINFRAME			128
+#define IDD_TESTAPPWIN32_DIALOG	102
+#define IDD_ABOUTBOX			103
+#define IDM_ABOUT				104
+#define IDM_EXIT				105
+#define IDI_TESTAPPWIN32			107
```

Comment:
In the Resource.h of both Playground and ValidationTests, we still have `#define`s that use TESTAPP in the name. Might consider renaming them to just APP or something.